vm/src/obj/objstr.rs

```diff
@@ -519,6 +519,88 @@ impl PyString {
         }
     }
 
+    // Return true if all characters in the string are printable or the string is empty,
```

Comment:
If you enhande these comments to the triple `///` format, the comment will be the python doc comment I believe.

Reply:
Thanks